ngraph/core/src/version.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/core/version.hpp"
+
+#include "ngraph/version.hpp"
+
+const char* NGRAPH_VERSION_NUMBER = CI_BUILD_NUMBER;
```

Comment:
I think we should align NGRAPH_VERSION_NUMBER with OPENVINO

Reply:
they are aligned. `CI_BUILD_NUMBER` is set as:
```
export CI_BUILD_NUMBER=2022.1.0-4696-fd875b97bd9-refs/pull/7600/head
```
And it's used in both ngraph, IE, OpenVINO runtime